.github/workflows/sycl-linux-precommit.yml

```diff
@@ -67,6 +67,7 @@ jobs:
             target_devices: ext_oneapi_level_zero:gpu;opencl:gpu;opencl:cpu
             reset_gpu: true
             install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
+	    extra_lit_opts: --param gpu-intel-gen12=True
```

Comment:
I actually don't know how it will affect cpu target. I'm not sure this parameter was designed for multi-target runs.

Reply:
I wouldn't expect anything weird given the usage of  `gpu-intel-gen12`. Probably in the general case of extra lit options there is a case where passing the same one to cpu and gpu is bad, but this one is pretty restricted so it's probably fine (?)

Reply:
@aelovikov-intel Do you know why Linux CI didn't run for this PR?

Reply:
No idea...

Reply:
@aelovikov-intel So what's the play, should we try to merge it as is given the CI passed and revert if there's some problem?

Reply:
I also don't know why post-commit task didn't run here

Reply:
Discussed offline, most likely an issue with the syntax in the workflow file.